src/app/drafts/containers/editor-container/editor-container.component.ts

```diff
@@ -43,18 +42,24 @@ export class EditorContainerComponent implements OnInit {
         )
       ),
       map(draft => ({ fields: omit(['id'], draft) })),
-      map(fields => createEditorConfig(fields))
+      withLatestFrom(this.currentUser$),
+      map(
+        ([fields, user]) =>
+          user
+            ? createEditorConfig(fields)
+            : createEditorConfig(merge(fields, { submitButtonDisabled: true }))
+      )
     );
   }
 
   updateDraft(post: SteeditorPost): void {
     this.store.dispatch(
-      fromFeatureStore.updateDraft({ ...post, id: this.currentDraft.id })
+      draftsActionCreators.updateDraft({ ...post, id: this.currentDraft.id })
     );
   }
 
-  broadcast(post: SteeditorPost): void {
-    this.store.dispatch(fromFeatureStore.broadcast(post));
+  broadcast(draft: Draft): void {
```

Comment:
It receives `SteeditorPost`. `broadcastDraft` action creator expects `Draft`, thus `id` should be added (like in `updateDraft`).